src/worker.py

```
@@ -573,12 +698,42 @@ def _is_basic_auth_valid(req, env) -> bool:
     for sql in _DDL:
         await env.DB.prepare(sql).run()
+    # Idempotent migration: add email_verified to existing users table.
+    # Fails silently when the column already exists (second run is a no-op).
+    try:
+        await env.DB.prepare(
+            "ALTER TABLE users ADD COLUMN email_verified INTEGER NOT NULL DEFAULT 0"
+        ).run()
+        # Pre-existing accounts are treated as verified (registered before this feature).
+        await env.DB.prepare("UPDATE users SET email_verified=1").run()
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)